MANIFEST.in

```diff
@@ -0,0 +1,11 @@
+# Exclude entire directories
+prune .github/
+prune dev_scripts/
+prune docs/
+prune examples/
+prune tests/
+
+# Exclude individual files
+exclude .* \
```

Comment:
I would rather accidentally include some useless files than missing some needed ones, so I should be really careful using wildcard for exclusion. Is there any hidden file needed?